webpack.config.dev.js

```diff
@@ -5,7 +5,7 @@ exports.entry.unshift('webpack/hot/dev-server');
 exports.module.loaders[0].loaders = ['babel', 'react-hot', 'jsx?harmony'];
 exports.plugins[1] = new webpack.DefinePlugin({
   __PROD__: false,
-  __S3_BUCKET__: 'landline'
+  __S3_BUCKET__: JSON.stringify('landline')
```

Comment:
We can't do that, though, because Landline doesn't know what the API URL is before its host initializes it (you can see this happening in `index.html`).